enable-secure-admin does not work with adopt-JDK 11

Calling enable-secure-admin brings up an Error on the next startup and asadmin does not work properly any more.

## Environment Details
GlassFish Version: 6.1
JDK version:  adopt jdk-11.0.11+9
OS: Debian 10, Win 10

----------

## Problem Description

Glassfish 6.1 seems to run with TLSv1: In adopt jdk-11.0.11+9 TLSv1 and TLSv1.1 are listed in the 'jdk.tls.disabledAlgorithms'-property in the 'java.security'-JDK-file. So asadmin can not establish a secure connetion to Glassfish. Removing TLSv1 and TLSv1.1 in  java.security nullifies the problem.

## Steps to reproduce
- run asadmin enable-secure-admin
- e.g. run asadmin  disable-secure-admin

## Impact of Issue
Glassfish can not be administered with the latest adopt 11 JDK (in a secure manner)

Stacktrace at startup:

```
[2021-07-22T15:53:10.650+0200] [glassfish 6.1] [WARNING] [NCLS-JMX-00007] [jakarta.enterprise.system.jmx] [tid: _ThreadID=192 _ThreadName=Thread-32] [timeMillis: 1626961990650] [levelValue: 900] [[
  Cannot start JMX connector JmxConnector config: { name = system, Protocol = rmi_jrmp, Address = 0.0.0.0, Port = 8686, AcceptAll = false, AuthRealmName = admin-realm, SecurityEnabled = true} due to exception java.io.IOException: Cannot bind to URL [rmi://user-PC:8686/jmxrmi]: javax.naming.CommunicationException [Root exception is java.rmi.ConnectIOException: error during JRMP connection establishment; nested exception is: 
	javax.net.ssl.SSLHandshakeException: No appropriate protocol (protocol is disabled or cipher suites are inappropriate)]]]

[2021-07-22T15:53:10.654+0200] [glassfish 6.1] [SEVERE] [] [] [tid: _ThreadID=192 _ThreadName=Thread-8] [timeMillis: 1626961990654] [levelValue: 1000] [[
  java.io.IOException: Cannot bind to URL [rmi://user-PC:8686/jmxrmi]: javax.naming.CommunicationException [Root exception is java.rmi.ConnectIOException: error during JRMP connection establishment; nested exception is: 
	javax.net.ssl.SSLHandshakeException: No appropriate protocol (protocol is disabled or cipher suites are inappropriate)]
	at java.management.rmi/javax.management.remote.rmi.RMIConnectorServer.newIOException(RMIConnectorServer.java:854)
	at java.management.rmi/javax.management.remote.rmi.RMIConnectorServer.start(RMIConnectorServer.java:501)
	at org.glassfish.admin.mbeanserver.RMIConnectorStarter.start(RMIConnectorStarter.java:296)
	at org.glassfish.admin.mbeanserver.JMXStartupService$JMXConnectorsStarterThread.startConnector(JMXStartupService.java:290)
	at org.glassfish.admin.mbeanserver.JMXStartupService$JMXConnectorsStarterThread.run(JMXStartupService.java:322)
Caused by: javax.naming.CommunicationException [Root exception is java.rmi.ConnectIOException: error during JRMP connection establishment; nested exception is: 
	javax.net.ssl.SSLHandshakeException: No appropriate protocol (protocol is disabled or cipher suites are inappropriate)]
	at jdk.naming.rmi/com.sun.jndi.rmi.registry.RegistryContext.rebind(RegistryContext.java:178)
	at java.naming/com.sun.jndi.toolkit.url.GenericURLContext.rebind(GenericURLContext.java:253)
	at java.naming/javax.naming.InitialContext.rebind(InitialContext.java:425)
	at java.naming/javax.naming.InitialContext.rebind(InitialContext.java:425)
	at java.management.rmi/javax.management.remote.rmi.RMIConnectorServer.bind(RMIConnectorServer.java:711)
	at java.management.rmi/javax.management.remote.rmi.RMIConnectorServer.start(RMIConnectorServer.java:496)
	... 3 more
Caused by: java.rmi.ConnectIOException: error during JRMP connection establishment; nested exception is: 
	javax.net.ssl.SSLHandshakeException: No appropriate protocol (protocol is disabled or cipher suites are inappropriate)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.createConnection(TCPChannel.java:300)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.newConnection(TCPChannel.java:196)
	at java.rmi/sun.rmi.server.UnicastRef.newCall(UnicastRef.java:343)
	at java.rmi/sun.rmi.registry.RegistryImpl_Stub.rebind(RegistryImpl_Stub.java:150)
	at jdk.naming.rmi/com.sun.jndi.rmi.registry.RegistryContext.rebind(RegistryContext.java:176)
	... 8 more
Caused by: javax.net.ssl.SSLHandshakeException: No appropriate protocol (protocol is disabled or cipher suites are inappropriate)
	at java.base/sun.security.ssl.HandshakeContext.<init>(HandshakeContext.java:170)
	at java.base/sun.security.ssl.ClientHandshakeContext.<init>(ClientHandshakeContext.java:98)
	at java.base/sun.security.ssl.TransportContext.kickstart(TransportContext.java:221)
	at java.base/sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:433)
	at java.base/sun.security.ssl.SSLSocketImpl.ensureNegotiated(SSLSocketImpl.java:829)
	at java.base/sun.security.ssl.SSLSocketImpl$AppOutputStream.write(SSLSocketImpl.java:1199)
	at java.base/java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:81)
	at java.base/java.io.BufferedOutputStream.flush(BufferedOutputStream.java:142)
	at java.base/java.io.DataOutputStream.flush(DataOutputStream.java:123)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.createConnection(TCPChannel.java:222)
	... 12 more
]]
```

--- a/nucleus/admin/util/src/main/java/com/sun/enterprise/admin/util/HttpConnectorAddress.java
+++ b/nucleus/admin/util/src/main/java/com/sun/enterprise/admin/util/HttpConnectorAddress.java
@@ -127,7 +127,7 @@ public final class HttpConnectorAddress {
     private SSLSocketFactory createAdminSSLSocketFactory(String alias, String protocol) {
         try {
             if (protocol == null) {
-                protocol = "TLSv1";
+                protocol = "TLS";
             }
             SSLContext cntxt = SSLContext.getInstance(protocol);
             /*

